src/main/java/uk/gov/hmcts/darts/cases/helper/AdvancedSearchRequestHelper.java

```diff
@@ -35,10 +41,13 @@
 
 @Component
 @SuppressWarnings({"PMD.TooManyMethods"})
+@RequiredArgsConstructor
 public class AdvancedSearchRequestHelper {
     @PersistenceContext
     private EntityManager entityManager;
```

Comment:
Should this be made `final` also for the RequiredArgsConstructor?

Reply:
Yep, updated entity manager to be final